python/mead/config/conll-single-tf.json

```diff
@@ -2,6 +2,7 @@
     "task": "tagger",
     "batchsz": 10,
     "conll_output": "conll-iobes-results.conll",
+    "modules":["preproc-exporters"],
```

Comment:
I wonder if we can think of a better way to make this an export-only concern

Reply:
we can always have a separate config file with this change, will that work?